### PR TITLE
Yo office teams toolkit

### PR DESCRIPTION
**Pull request by Rick-Kirkham**
DO NOT MERGE UNTIL AROUND MARCH 29TH.

### CODE REVIEW

File: docs/develop/teams-toolkit-overview.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    ![Screenshot showing three options in Teams Toolkit option drop down. The second option "Start with an Outlook add-in" is selected.](../images/teams-toolkit-create-outlook-add-in.png)
+
+1. In the **Select a capability** drop down, select **Outlook Taskpane Add-in (preview)**.
+
+    ![Screenshot showing two options in Teams Toolkit capability drop down. The first option "Outlook Taskpane Add-in (preview)" is selected.](../images/teams-toolkit-create-outlook-task-pane-capability.png)
+
+1. In the **Workspace folder** dialog that opens, select the folder where you want to create the project.
+1. Give a name to the project (with no spaces) when prompted. Teams Toolkit will create the project with basic files and scaffolding. It will then open the project *in a second Visual Studio Code window*. Close the original Visual Studio Code window.
+1. In the Visual Studio Code **TERMINAL** navigate to the root of the project and run `npm install`.
+1. Before you make changes to the project, verify that Outlook add-in sideloading from Visual Studio Code works. Use the following steps:
```
Comment (AlexJerabek) [inline docs/develop/teams-toolkit-overview.md:35]: Small change to make the behavior of the sideloading seem less random. 
```suggestion
1. Before you make changes to the project, verify that you can sideload your Outlook add-in sideloading from Visual Studio Code. Use the following steps:
```

File: docs/develop/teams-toolkit-overview.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+    <ol type="a">
+      <li>Ensure that your account in your Microsoft 365 developer tenancy is also an email account in desktop Outlook. If it isn't, follow the guidance in <a href="https://support.microsoft.com/office/add-an-email-account-to-outlook-e9da47c4-9b89-4b49-b945-a204aeea6726">Add an email account to Outlook</a>.</li>
+      <li><i>Close Outlook desktop.</i></li>
```
Comment (AlexJerabek) [inline docs/develop/teams-toolkit-overview.md:39]: Why is this italicized?

File: docs/develop/teams-toolkit-overview.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      <li>In Visual Studio Code, open the Teams Toolkit.</li>
```
Comment (AlexJerabek) [inline docs/develop/teams-toolkit-overview.md:40]: Sometimes in this article, it's referred to "the Teams Toolkit" and others it's just "Teams Toolkit". Minor thing, but something to consider standardizing. I like "the Teams Toolkit: more, FWIW.

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
Looks good @Rick-Kirkham. I have some minor feedback, but also want to clarify that I can't test this yet, right? I have the pre-release version of Teams Toolkit, but don't see the Outlook Add-in option.

File: docs/develop/teams-toolkit-overview.md (hunk 2 above)
Comment (Rick-Kirkham) [reply docs/develop/teams-toolkit-overview.md:39]: It's something that is easy to overlook and really important, but I'll remove the italics.

File: docs/develop/teams-toolkit-overview.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+      <li>Select <b>View</b> | <b>Run</b> in Visual Studio Code. In the <b>RUN AND DEBUG</b> drop down menu, select the option, <b>Outlook Desktop (Edge Chromium)</b>, and then press F5. The project builds and a Node dev-server window opens. This process may take a couple of minutes. Eventually, Outlook desktop will open.</li>
+      <li>Open the <b>Inbox</b> <i>of your Microsoft 365 account identity</i> and open any message. A <b>Contoso Add-in</b> tab with two buttons will appear on the <b>Home</b> ribbon (or the <b>Message</b> ribbon, if you have opened the message in its own window).</li>
+      <li>Click the <b>Show Taskpane</b> button and a task pane opens. Click the <b>Perform an action</b> button and a small notification appears near the top of the message.</li>
+      <li>To stop debugging and uninstall the add-in, select <b>Run</b> | <b>Stop Debugging</b> in Visual Studio Code.</li>
+    </ol>
+
+Now you can change and develop the project. In places where the guidance in the Office Add-ins documentation branches depending on what type of manifest is being used, be sure to follow the guidance for the unified manifest.
```
[the remark this answers is not in the document]
Comment (Rick-Kirkham) [reply docs/develop/teams-toolkit-overview.md:48]: We don't do that for the Yo Office article, so I'm inclined to keep this one parallel. Also, this isn't a quickstart. It's for people who haven't used the tool before, but not for people that are completely new to add-ins. Presumably (as for the Yo Office article) they understand where they need to go next to start implementing their add-in.

File: docs/develop/teams-toolkit-overview.md (hunk 2 above)
Comment (AlexJerabek) [reply docs/develop/teams-toolkit-overview.md:39]: Maybe bold it then? Something about the whole line being italicized makes it seem _less_ important. Almost like it's a footnote.

File: docs/develop/teams-toolkit-overview.md (hunk 3 above)
Comment (Rick-Kirkham) [reply docs/develop/teams-toolkit-overview.md:40]: You're right that we should be consistent, and I share your preference for including the "the". But I just looked at the Teams docs and they don't use the "the", so for consistency across both doc sets, I removed the "the"s.

**Comment by Rick-Kirkham** [comment]
> Looks good @Rick-Kirkham. I have some minor feedback, but also want to clarify that I can't test this yet, right? I have the pre-release version of Teams Toolkit, but don't see the Outlook Add-in option.

I think that is expected. This won't publish until around March 29th. I'll send you the instructions for the bug bash tomorrow in an email. I don't want to put that link in a public place like this.

**Review by AlexJerabek — APPROVED** [review]
Thanks @Rick-Kirkham! Looks good.

**Review by lindalu-MSFT — APPROVED** [review]
Looks good to me

**Pull request merged**